peps/pep-0772.rst

```diff
@@ -0,0 +1,455 @@
+index maintainers. With greater transparency and clearer goals via an elected
+Packaging Council, the PSF will also be able to offer more tactical work,
+longer term strategic vision, and fundraising support to all parts of this
+ecosystem.
+
+Some expectations around the formation of a Packaging Council are that they
+will facilitate:
+
+* Work with PSF staff and the newly formed `User Success Workgroup`_ toward
+  improving the user experience around packaging.
+* A more stable packaging ecosystem that is more responsive to community input.
+* Increased transparency and clearer shared goals for the packaging ecosystem.
+* Increased tactical and fundraising support from the PSF, to increase capacity
+  and funding available to packaging tools.
+
+=============
+Specification
+=============
+
+-----------------
+Packaging Council
+-----------------
+
+The Packaging Council will be composed of 5 individuals.
+
+Mandate
+=======
+
+The council shall work to:
+
+* Improve Python packaging's user experience.
+* Maintain the quality and stability of the Python packaging standards.
+* Formalise and maintain the relationship with the core team as well as the
+  PSF.
+* Establish appropriate decision-making processes.
+* Seek consensus among contributors before acting in a formal capacity.
+* Make contributing as accessible, inclusive, and sustainable as possible.
+
+Responsibilities
+================
+
+The council shall:
+
+* have broad authority over the Python packaging standards and Python Packaging
+  User Guide, that are maintained on packaging.python.org.
+* establish processes for making binding decisions regarding packaging
+  standards, tools and implementations as well as for considering
+  ecosystem-wide changes.
+* look for ways to use their direct authority as little as possible – seeking
+  consensus/consent over voting.
+
+To use its powers, the council votes. Every council member must either vote or
+explicitly abstain. Members with conflicts of interest on a particular vote
+must abstain. Passing requires support from a majority of non-abstaining
+council members.
+
+Whenever possible, the council is expected to share its decisions and processes
+publicly in a timely fashion.
+
+Delegations
+-----------
+
+The Python Steering Council would be expected to delegate decision making to
+the Packaging Council for PEPs related to the Python packaging.  The bodies
+would work together on issues that intersect the packaging domain and language
+stewardship (including the CPython implementation, standard library, and
+distribution).
+
+The PSF Board would be encouraged to formally deprecate the Packaging Workgroup
+and the Packaging Council would wholly take on the responsibilities of the
+PSF's Packaging Workgroup.
+
+The PyPA would be expected to work with the Packaging Council to establish a
+decision making process that governs the technical projects under the PyPA
+umbrella.
+
+Processes
+=========
```

Comment:
The process set out in this section seems very "heavy", for want of a better word. Could you add some text to Rejected Ideas for why a [PEP 729](https://peps.python.org/pep-0729/#operations-and-process)-like approach wasn't adopted?